piptools/scripts/compile.py

```diff
@@ -8,20 +8,21 @@
 import tempfile
 
 import pip
+
+# Make sure we're using a compatible version of pip
+from ..utils import assert_compatible_pip_version
+assert_compatible_pip_version()
```

Comment:
There is still a possibility of error if an `ImportError` is thrown in `utils.py`, but the fix for that is more destructive, and would mean moving this function into its own file. I don’t mind doing that, but I’ll wait to see if reviewers think that would be necessary.